jenkins.groovy

```diff
@@ -65,6 +65,8 @@ def clearImages() {
 
 def buildStrimzi() {
     sh "mvn clean install -DskipTests"
+//  Prepare third-party libs fro kafka docker image.
+    sh "mvn dependency:copy-dependencies -pl kafka-thirdparty-libs -DoutputDirectory=../docker-images/kafka/thirdparty-libs"
```

Comment:
This will build it only on Jenkins. We need to add this somewhere to the `docker-images/build.sh` or to one of the `Makefile` files in `docker-images` or in `docker-images/kafka`. Can you suggest the best place @tombentley? You probably know this area best.

Reply:
Before the `touch` in the `.kafka-agent.tmp` goal of `docker-images/kafka/Makefile` should do it, I think.

Reply:
@tombentley @scholzj please check current version?